.github/workflows/ci.canary.yml

```diff
@@ -135,15 +135,15 @@ jobs:
           yarn build
           yarn workspace @superfluid-finance/ethereum-contracts test-coverage
 
-      # - name: Install lcov
-      #   run: sudo apt-get -y install lcov
-
-      # - name: Clean up and merge coverage artifacts
-      #   run: |
-      #     # extract coverage for NFT contracts from forge coverage
-      #     lcov -e lcov.info -o lcov.info 'packages/ethereum-contracts/contracts/superfluid/*NFT*.sol'
-      #     # merge hardhat and forge coverage files
-      #     lcov -a lcov.info -a packages/ethereum-contracts/coverage/lcov.info -o packages/ethereum-contracts/coverage/lcov.info
+      - name: Install lcov
```

Comment:
will it work?

Reply:
did another test here: https://github.com/superfluid-finance/protocol-monorepo/actions/runs/4531480256/jobs/7981669257
previously spent a long time trying to get it to work, but just wanted to double check in ci.feature above